pkg/sshd/server.go

```diff
@@ -215,6 +223,19 @@ func (s *server) answer(channel ssh.Channel, requests <-chan *ssh.Request, sshCo
 					req.Reply(ok, nil)
 					break
 				}
+
+				repoName := parts[1]
```

Comment:
can you ensure that `len(parts) >= 2`. I know that it should in the common case, but it would be good to be defensive